cdn-uploader.js

```diff
@@ -0,0 +1,41 @@
+const ftp = require("basic-ftp");
+
+const config = {
+	host: process.env.AKAMAI_CDN_HOSTNAME,
+	user: process.env.AKAMAI_CDN_USERNAME,
+	password: process.env.AKAMAI_CDN_PASSWORD,
+	cpCode: process.env.AKAMAI_CDN_CPCODE,
+	port: 21,
```

Comment:
could you remove the trailing comma? is _legal_ now, but ugly :P

Reply:
Removed :)